core/init.sh

```diff
@@ -0,0 +1,11 @@
+
+#!/usr/bin/env bash
+
+echo "Hack SLOTS_IN_EPOCH in lodestar"
+cd packages/test && ./scripts/hack-ethereum.sh && cd -
+
+echo "Install husky hook"
+cd .. && ./core/node_modules/.bin/husky install && cd -
+
+echo "Initialize foundry libraries"
+cd packages/contracts && (forge install||true) && cd -
```

Comment:
Similar to Vincent's comment in flake.nix, would subshells also work here?
```suggestion
echo "Hack SLOTS_IN_EPOCH in lodestar"
(cd packages/test && ./scripts/hack-ethereum.sh)

echo "Install husky hook"
(cd .. && ./core/node_modules/.bin/husky install)

echo "Initialize foundry libraries"
(cd packages/contracts && (forge install||true))
```

Reply:
Exactly!